tools/snp.sh

```diff
@@ -646,9 +663,16 @@ set_acl_for_sev_device() {
   echo "${setfacl_command}" | sudo tee -a "${rc_local_file}" >/dev/null
 }
 
-build_and_install_amdsev() {
+build_and_install_amdsev() {  
+  
   local amdsev_branch="${1:-${AMDSEV_DEFAULT_BRANCH}}"
-
+  
+  if [ "$SVSM" = true ]; then
+    AMDSEV_URL=${AMDSEV_SVSM_URL}
+    amdsev_branch="svsm-latest-fixes"    	 
```

Comment:
You should probably use the global value here, as you already defined this string at the top?